index.html

```diff
@@ -20,12 +20,123 @@
         
     </header>
 
-<section id="home_section">
+<section id="work_section">
+    <section id="home_section">
+        <div class="home_container">
+        <h1>I’m Shahnawaz Ali
+            Glad to see you!</h1>
+    
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <span>LET’S CONNECT</span>
+    
+                <ul class="social_media">
+                    <li><a href="https://twitter.com/RjShahnawaza75"><img src="./images/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/shahnawaz-ali-a24b72204" ><img src="./images/Linkedin_icon.svg" alt="linkedin"></a></li>
+                    <li><a href="#"><img src="./images/images4.svg" alt="ok"></a></li>
+                    <li><a href="#"><img src="./images/images5.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
+    
+                </ul>
+            </div>
+    </section>
```

Comment:
- [ ] The headline section should be separated from the work section and not a section in the work section. Kindly take the headline section out of the work section as this will improve your webpage SEO.

Reply:
Thank you for the feedback. My header is already separated from the home_section . Could you elaborate more please so that i can fix it.

Reply:
The way you have it set up, the headline is a section in the work section container. the work section container should come after the headline section container.